README.md

```diff
@@ -39,28 +39,87 @@ Same as the original conductor, the primary goals of this project are to...
 - Provide a free to use framework for any starting enterprise, or individual programmer.
 - Automatic detection of connected iOS devices
 
-# Default Properties
+# Configuration
+Conductor can be configured using a [yaml](https://en.wikipedia.org/wiki/YAML) file. By default, Conductor looks for a "config.yaml" at the root of embedded resources.
+
+The file has 3 sections: `current configuration`, `defaults`, and `schemes`.
+
+## General Configuration
+The general configuration only has two properties: `platformName` (which must be either IOS or ANDROID) and optionally `currentSchemes`, which will be discussed later
+
+## Defaults
+The defaults section contains both general defaults and defaults per platform. It looks like this:
+```yaml
+defaults:
+  retries: 3
+  timeout: 10
+  screenshotOnFail: false
+  autoGrantPermissions: true
+
+  ios:
+    platformVersion: 11.0
+    deviceName: iPhone 8
+
+  android:
+    timeout: 15
+    platformVersion: 8.0
+```
+Platforms can override defaults just be re-specifying them in the platform section
+
+## Schemes
+
+Sometimes it is useful to specify properties under specific circumstances, and this is what "schemes" are for.  Schemes override properties in the configuarion *in the order they are specified in the `currentSchemes` section of the configuration file.*
+
+You might, for example, have a scheme for running on a specific device or specific remote testing tool. It's a pain to have to re-specify these so, Conductor makes this easy with schemes. Some example schemes:
+
+```yaml
+ios_sauce_labs:
+  hub: http://saucelabs-hub
+  appFile: sauce-storage:app.zip
+  automationName: XCUITest
+  platformVersion: 11.1
+  deviceName: iPhone 8
+
+shorter_timeouts:
+  timeout: 1
+  retries: 2
+```
+You can see a variety of example configuration files in the unit tests for conductor [here](src/test/resources/test_yaml)
+
+# Supported Properties
+## General (Common)
+- `platformName` = {string: ANDROID or IOS} (note all capps)
```

Comment:
Do we still need this one?

Reply:
Yes it's still technically a supported property.